lms/djangoapps/verify_student/views.py

```diff
@@ -895,8 +895,8 @@ def _compose_message_reverification_email(
         reverification_block = modulestore().get_item(usage_key)
         # Allowed attempts is 1 if not set on verification block
         allowed_attempts = 1 if reverification_block.attempts == 0 else reverification_block.attempts
-        user_attempts = VerificationStatus.get_user_attempts(user_id, course_key, relates_assessment, location_id)
-        left_attempts = allowed_attempts - user_attempts
+        used_attempts = VerificationStatus.get_user_attempts(user_id, course_key, relates_assessment, location_id)
+        left_attempts = allowed_attempts - used_attempts
```

Comment:
@wedaly I have some confusion about
 `allowed_attempts = reverification_block.attempts + 1`  this will work fine only when author has not set attempts otherwise it will increase user attempts by 1.